frontend/src/components/dashboard/campaigns/announcement-modal/AnnouncementModal.module.scss

```diff
@@ -0,0 +1,57 @@
+@import 'styles/_variables';
+@import 'styles/_mixins';
+
+$modal-width: $tablet;
+$title-height: $height-4;
+.modalBg {
+  @include flex-center();
+  background: adjust-color($secondary, $alpha: -0.05);
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  z-index: 120;
+  top: 0;
+  left: 0;
+}
+
+.modal {
+  max-width: 100vw;
+  border-radius: 0.6rem;
+  background-color: $neutral-light;
+  height: auto;
+  max-height: calc(70vh);
+  position: relative;
+  overflow: hidden;
+  overflow-y: auto;
+  padding: 2rem;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+
+  @include mobile {
+    border-radius: 0;
+    min-height: 100vh;
+  }
+
+  @include scrollbar($neutral-light, $neutral);
+}
+
+.modalImg {
+  max-height: calc(50vh);
+}
+
+.content {
+  max-width: 100vw;
+  margin: 32px 0px;
+  text-align: center;
+}
+
+.close {
+  position: absolute;
+  top: 2rem;
+  right: 2rem;
+  &.modalTitleClose {
+    color: $green-900;
+    top: calc(#{$title-height} / 2 - 1.5rem); // close button has height of 3rem
+  }
+}
```

Comment:
This styling is... janky. I mainly tried to follow what was in the figma - but I realise we may have changed since then because of the new design system. Please let me know if there's something better I should model after!

Reply:
I think desktop looks ok. Might want to fix mobile though.

<img src="https://user-images.githubusercontent.com/3666479/102946093-965b2300-44fa-11eb-8f05-9c25cfe499e0.png" width=300 />